### ISSUE TITLE
Add a Help menu item

### ISSUE THREAD

**Issue by mwes** [SD2E/experimental-intent-parser#71]
In GitLab by @nwalczak on Jul 3, 2019, 10:18

It would be good to have some documentation to help users out.  We could also use it to print version information for the script, to help ensure users are using the proper version.

**Comment by mwes** [comment]
In GitLab by @nwalczak on Jul 3, 2019, 10:18

Here's the current text I have in mind:

```
<p>
Intent Parser version ${VER_STRING}
</p
<p>
The purpose of the intent parser add-on is to create a suite of tools that aids the user in creating well documented experimental plans.  
</p>

<p>
Right now, there are three different actions that can be performed to help the user.  The first action is <i>analyzing</i> the document, which searches for terms present in the SD2 spreadsheet dictionary and offers to insert links to the SynbioHub definition of those terms.  Additionally, the document can be scanned for terms that don't exist in the <i>spelling</i> dictionary and suggest them as terms to possibly add into the SD2 spreadsheet dictionary or to have a link manually added.  Users can also highlight arbitrary terms and <i>add</i> a definition to SynbioHub for that term.  This <i>add</i> dialog will also query SynbioHub for matches to the selected term and links to those terms can be added.
</p>
```

**Comment by mwes** [comment]
In GitLab by @nwalczak on Jul 3, 2019, 11:44

created branch [`70-add-a-help-menu-item`](https://gitlab.sd2e.org/sumorok/experimental-intent-parser/compare/develop...70-add-a-help-menu-item)

**Comment by mwes** [comment]
In GitLab by @nwalczak on Jul 3, 2019, 11:44

mentioned in merge request !23

**Comment by mwes** [comment]
In GitLab by @nwalczak on Jul 3, 2019, 13:48

closed via merge request !23

**Comment by mwes** [comment]
In GitLab by @nwalczak on Jul 3, 2019, 13:48

closed via commit 372c06fcea7b588a15f44c59ff5f5a5258e03e01

**Comment by mwes** [comment]
In GitLab by @nwalczak on Jul 3, 2019, 14:26

mentioned in commit 372c06fcea7b588a15f44c59ff5f5a5258e03e01
